Desenvolvimento

CrystianSantos opened the pull request.

Rynilan committed:
docs: atualizado o readme.

Rynilan committed:
docs: adicionado o .gitignore contendo a informação para a pasta __pycache__ e arquivos binários do python, .pyc.

docs: criada a estrutura de pastas.

Rynilan committed:
docs: adicionados os créditos aos atuais colaboradores.

Rynilan committed:
docs: corrigido erro no readme.

Rynilan committed:
docs: adicionada descrição de importância astronômica.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
docs: adicionados alguns membros aos créditos.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
docs: adicionado um membro aos créditos.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
docs: atualizado o readme

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: adicionadas as classes abstratas

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Mps192004 committed:
arquivo serviços (#1)

* arquivo serviços

* code: ajustada a identação e a herança (__super__())

---------

Co-authored-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: criado o programa primcipal.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

CrystianSantos committed:
Criação da classe barbearia

eu criei as bases das funções dentro da classe barbearia

Rynilan committed:
code: ajustada a arvore de diretórios da pasta assets.

code: feitas alterações nas classes que seriam abstratas.
Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

CrystianSantos committed:
finalização da classe barbearia

foram terminadas

CrystianSantos committed:
conclusão das classes

foram terminadas as minhas classes, tanto da barbearia quanto do agendamento

CrystianSantos committed:
Merge branch 'desenvolvimento' of https://github.com/Rynilan/Connected-Barber into desenvolvimento

CrystianSantos committed:
exclui uma função que não era minha

exclui a classe cliente que havia criado que não pertence a mim

DRXMORENO committed:
produtos (#3)

uma classe que obtém herança de oferecimentos

CrystianSantos committed:
Desenvolvimento (#2)

* docs: atualizado o readme.

* docs: adicionado o .gitignore contendo a informação para a pasta __pycache__ e arquivos binários do python, .pyc.

docs: criada a estrutura de pastas.

* docs: adicionados os créditos aos atuais colaboradores.

* docs: corrigido erro no readme.

* docs: adicionada descrição de importância astronômica.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

* docs: adicionados alguns membros aos créditos.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

* docs: adicionado um membro aos créditos.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

* docs: atualizado o readme

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

* code: adicionadas as classes abstratas

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

* arquivo serviços (#1)

* arquivo serviços

* code: ajustada a identação e a herança (__super__())

---------

Co-authored-by: Rynilan <ilanvitorsantos@gmail.com>

* code: criado o programa primcipal.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

* Criação da classe barbearia

eu criei as bases das funções dentro da classe barbearia

* code: ajustada a arvore de diretórios da pasta assets.

code: feitas alterações nas classes que seriam abstratas.
Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

* finalização da classe barbearia

foram terminadas

* conclusão das classes

foram terminadas as minhas classes, tanto da barbearia quanto do agendamento

* exclui uma função que não era minha

exclui a classe cliente que havia criado que não pertence a mim

* produtos (#3)

uma classe que obtém herança de oferecimentos

---------

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>
Co-authored-by: Rynilan <ilanvitorsantos@gmail.com>
Co-authored-by: Mps <melissaguimaraes442@gmail.com>
Co-authored-by: Paulo Gabriel da luz <paulogp4319@gmail.com>

21Lu1z committed:
criação da classe cliente

Criei uma classe cliente que recebe as funções cep e preferencias

21Lu1z committed:
Merge branch 'desenvolvimento' of https://github.com/Rynilan/Connected-Barber into desenvolvimento

LuizAntonioLz committed:
Class Admin

Criação da classe admin e funções iniciais do admin

LuizAntonioLz committed:
Merge branch 'desenvolvimento' of https://github.com/Rynilan/Connected-Barber into desenvolvimento

21Lu1z committed:
Add files via upload (#4)

criei uma classe com as funções de cep e preferencias

Rynilan committed:
code: criada a estrutura base do banco de dados.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: removida linha 'drop database Connected_Barber' (apenas para desenvolvimento do banco.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
Merge github.com:Rynilan/Connected-Barber
equalização de branchs

Rynilan committed:
Merge branch 'desenvolvimento' of github.com:Rynilan/Connected-Barber

Rynilan committed:
code: ajuste da classe de administrador.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: adicionados métodos necessários à barbearia.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: ajustada a classe cliente, com adição dos parâmetros necessários da herança, privação de atributos e adição de métodos que virão a ser documentados.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: ajustada a classe ator.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: adicionados métodos necessários para a classe oferecimento.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: ajustada a classe produto, corrigida sintaxe, adicionados atributos singulares e métodos getters e setters.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: ajustada a classe de serviços.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: esses arquivos só foram movidos para a pasta assets.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: ajustada a classe endereço, adição dos setters, ligação direta com uma barbearia e adição do atributo complemento.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: adicionado o campo data à tabela de agendamentos.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: reajustada a classe barbearia.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: refinada a forma de verificação de permissão dos metodos setters da classe endereço.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: ajustadas as relações da classe cliente

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

LuizAntonioLz committed:
Class Adm

Produção das funções da classe de administrador

JosiasAragao committed:
Add files via upload

objeto funcionario

Rynilan committed:
code: ajustada a estrutura do banco de dados

agora a chave primaria da barbearia é seu nome (não foi testado)

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: ajustado a estrutura do banco, testado e agora sem bugs.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: adicionada as funções de conexão ler e atualizar dados do banco da tabela_barbearia, falta teste.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: ajustado o README.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: ajustado o escopo da classe endereço.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: ajustada a estrutura do banco de dados, simplificada.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: ajustada a estrutura do banco de dados.

Mps192004 committed:
mais um arquivo

create função do CRUD no banco  de dados

Rynilan committed:
code: pequenos ajustes.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: pequenos ajustes.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
docs: atualizado o readme.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

DRXMORENO committed:
Add files via upload

CrystianSantos committed:
coneção e o crud

foram feitas as conexões com o banco por vias das dúvidas e o crud para a variável Endereço

Mps192004 committed:
mais um arquivo chato

Rynilan commented:
Crystian, não há descrição ou commit que diferencie seu repositório do atual, foi este erro ou um teste?

Rynilan committed:
code: organizado todo o CRUD da tabela barbearia em sua respectiva pasta.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: organizado o crud da tabela usuário em sua respectiva pasta

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: organizados os cruds das respectivas tabelas em cada arquivo

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
docs: adicionados avisos internos a cada pasta referente a tabelas com funções pendentes.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: transformada arquivo em função.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: ajustado o crud da tabela endereço.

removido alguns prints, ajustada a formação e a função de atualizar.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>

Rynilan committed:
code: ajustado crud da tabela endereço.

modificado o nome das funções para combinar com o comando sql respectivo, transformada a classe em funções, ajustados erros de digitação.

Signed-off-by: Rynilan <ilanvitorsantos@gmail.com>
